gnu.elf/gnu.elf.properties

```diff
@@ -16,9 +16,11 @@ buildArtifactType.emsk = ARC EM Starter Kit Projects
 buildArtifactType.iotdk = ARC IoT Development Kit Projects
 buildArtifactType.hsdk = ARC HS Development Kit Projects
 buildArtifactType.emsdp = ARC EM SDP Projects
+buildArtifactType.hs6xdk = ARC HS6x Development Kit Projects
```

Comment:
There is no "HX6x Development Kit", the toolchain should attach itself to regular `buildArtifactType.application` projects only.